TASVideos/Services/WikiToTextRenderer.cs

```diff
@@ -53,10 +66,34 @@ public bool CheckCondition(string condition)
 				return result;
 			}
 
-			public Task RunViewComponentAsync(TextWriter w, string name, IReadOnlyDictionary<string, string> pp)
+			public async Task RunViewComponentAsync(TextWriter w, string name, IReadOnlyDictionary<string, string> pp)
 			{
-				// TODO: This needs to be handled in concert with Wiki changes
-				return Task.CompletedTask;
+				var componentExists = ModuleParamHelpers.TextComponents.TryGetValue(name, out Type? textComponent);
+				if (!componentExists)
+				{
+					return;
+				}
+
+				var invokeMethod = textComponent!.GetMethod("RenderTextAsync");
+
+				if (invokeMethod == null && textComponent.GetMethod("RenderText") != null)
+				{
+					throw new NotImplementedException("Sync method not supported yet");
```

Comment:
How strongly do we feel about also supporting a `RenderText`(Sync)?  I don't see any problem with doing just async and cutting this entirely.